docs/verifiers-guide/deployment.md

```diff
@@ -0,0 +1,172 @@
+  service.
+- Docker. Used to run CKB full node and indexer.
+- CKB RPC and indexer endpoint. Recommend to run a full node via docker locally. 
+- Ethereum RPC endpoint. Recommend to use [infura](https://infura.io/dashboard/ethereum) service.
+
+```bash
+# install forcecli
+$ npm install -g @force-bridge/cli@latest
+
+# check if it works
+$ forcecli -h
+Usage: forcecli [options] [command]
+
+forcecli is command line tool to lock & unlock asset to force bridge
+
+Options:
+  -V, --version        output the version number
+  -h, --help           display help for command
+
+Commands:
+  eth
+  collector [options]
+  fee                  query and withdraw bridge fee
+  rpc [options]
+  verifier [options]
+  config
+  keystore
+  help [command]       display help for command
+
+
+# run CKB full node and indexer
+# reference: <https://hub.docker.com/r/nervos/perkins-tent>
+# you can change the port and mount directory as you want.
+$ docker run -d -it -p 8117:9115  --name=ckb-testnet-indexer  -e "CKB_NETWORK=testnet" -v /path/to/ckb/data:/data nervos/perkins-tent:v0.43.0
+
+# check if it works
+echo '{
+    "id": 2,
+    "jsonrpc": "2.0",
+    "method": "get_blockchain_info",
+    "params": []
+}' \
+| tr -d '\n' \
+| curl -H 'content-type: application/json' -d @- \
+http://localhost:8117/rpc
+
+echo '{
+    "id": 2,
+    "jsonrpc": "2.0",
+    "method": "get_cells",
+    "params": [
+        {
+            "script": {
+                "code_hash": "0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8",
+                "hash_type": "type",
+                "args": "0x8211f1b938a107cd53b6302cc752a6fc3965638d"
+            },
+            "script_type": "lock"
+        },
+        "asc",
+        "0x64"
+    ]
+}' \
+| tr -d '\n' \
+| curl -H 'content-type: application/json' -d @- \
+http://localhost:8117/indexer
+```
+
+## Generate Private Key and Associated Configs
+
+Generate your own private key. The private key is used to sign the cross chain transactions, keep it safe.
+
+```bash
+# generate the multiConfig
+$ export PRIVKEY=0x1000000000000000000000000000000000000000000000000000000000000000
+$ forcecli config generate -k $PRIVKEY
+multiConfig:
+{
+  "ethAddress": "0x7B2419E0Ee0BD034F7Bf24874C12512AcAC6e21C",
+  "ckbPubkeyHash": "0x42a34b11710e40b97180f8edae2760c0ab69bcf3",
+  "ckbAddress": "ckt1qyqy9g6tz9csus9ewxq03mdwyasvp2mfhnesju9rhz"
+}
+# Send the multiConfig to the collector, it will be used in the deployment of on-chain contracts. 
+
+
+# generate keystore
+$ echo "{\"force-bridge\":\"${PRIVKEY}\"}" > keys.json
+$ forcecli keystore encrypt -s ./keys.json -d ./keystore.json -p my-custom-password
+# 1. Save the keystore.json file and remember your password.
+# 2. Delete the keys.json file, close the terminal and clear the shell commands history to keep your private key safe.
+```
+
+## Run Verifier Service
+
+When we gets the multisig config from all verifiers, we will deploy the contracts, and then send verifiers a service
```

Comment:
```suggestion
When we gets the multisig config from all verifiers, we will deploy the contracts, and then run verifier  as a service
```

Reply:
it means we will send the verifiers the config, not run the service ourselves